apisix/plugins/ldap-auth.lua

```diff
@@ -100,6 +100,11 @@ local function extract_auth_header(authorization)
 end
 
 function _M.rewrite(conf, ctx)
+    if not ok then -- ensure rasn library loaded
```

Comment:
For code style, we return 501 in the other similar cases:
https://github.com/apache/apisix/blob/7e292eadd36e738f84647634b0c351bc268ea529/apisix/plugins/gzip.lua#L104-L108

Reply:
changed